pkg/services/orguser/model.go

```diff
@@ -0,0 +1,21 @@
+package orguser
+
+import "time"
+
+type OrgUser struct {
+	ID      int64 `xorm:"pk autoincr 'id'"`
+	OrgID   int64 `xorm:"org_id"`
+	UserID  int64 `xorm:"user_id"`
+	Role    RoleType
+	Created time.Time
+	Updated time.Time
+}
+
+// swagger:enum RoleType
```

Comment:
Question f(or my other PR: do we need swagger directives in the models generally, or is this a special case?